Bindings/Java/Matlab/Dynamic_Walking_Challenge/Dynamic_Walker_Builder/buildWalker_Complete.m

```diff
@@ -0,0 +1,456 @@
+osimModel.addContactGeometry(groundContactSpace);
+
+
+% Make a Right Hip Contact
+RightHipContactSphere = ContactSphere();
+RightHipContactSphere.setRadius(contactSphereRadius);
+RightHipContactSphere.setLocation( Vec3(0,0,pelvisWidth) );
+RightHipContactSphere.setFrame(pelvis)
+RightHipContactSphere.setName('RHipContact');
+osimModel.addContactGeometry(RightHipContactSphere);
+
+
+% Make a Left Hip Contact
+LeftHipContactSphere = ContactSphere();
+LeftHipContactSphere.setRadius(contactSphereRadius);
+LeftHipContactSphere.setLocation( Vec3(0,0,-pelvisWidth) );
+LeftHipContactSphere.setFrame(pelvis)
+LeftHipContactSphere.setName('LHipContact');
+osimModel.addContactGeometry(LeftHipContactSphere);
+
+
+% Make a Right Knee Contact
+RightKneeContactSphere = ContactSphere();
+RightKneeContactSphere.setRadius(contactSphereRadius);
+RightKneeContactSphere.setLocation( Vec3(0,thighLength/2,0) );
+RightKneeContactSphere.setFrame(rightShank)
+RightKneeContactSphere.setName('RKneeContact');
+osimModel.addContactGeometry(RightKneeContactSphere);
+
+% Make a Left Knee Contact
+LeftKneeContactSphere = ContactSphere();
+LeftKneeContactSphere.setRadius(contactSphereRadius);
+LeftKneeContactSphere.setLocation(Vec3(0,thighLength/2,0));
+LeftKneeContactSphere.setFrame(leftShank)
+LeftKneeContactSphere.setName('LKneeContact');
+osimModel.addContactGeometry(LeftKneeContactSphere);
+
+% Make a Right Foot Contact Sphere
+RightFootContactSphere = ContactSphere();
+RightFootContactSphere.setRadius(contactSphereRadius);
+RightFootContactSphere.setLocation( Vec3(0,-shankLength/2,0) );
+RightFootContactSphere.setFrame(rightShank)
+RightFootContactSphere.setName('RFootContact');
+osimModel.addContactGeometry(RightFootContactSphere);
+
+% Make a Left Foot Contact Sphere
+LeftFootContactSphere = ContactSphere();
+LeftFootContactSphere.setRadius(contactSphereRadius);
+LeftFootContactSphere.setLocation( Vec3(0,-shankLength/2,0) );
+LeftFootContactSphere.setFrame(leftShank)
+LeftFootContactSphere.setName('LFootContact');
+osimModel.addContactGeometry(LeftFootContactSphere);
+
+% **********  END CODE  **********
+
+% Define Contact Force Parameters
+stiffness           = 1000000;
+dissipation         = 2.0;
+staticFriction      = 0.8;
+dynamicFriction     = 0.4;
+viscousFriction     = 0.4;
+transitionVelocity  = 0.2;
+
+% Make a Hunt Crossley Force for Right Hip and update parameters
+HuntCrossleyRightHip = HuntCrossleyForce();
+HuntCrossleyRightHip.setName('RHipForce');
+HuntCrossleyRightHip.addGeometry('RHipContact');
+HuntCrossleyRightHip.addGeometry('PlatformContact');
+HuntCrossleyRightHip.setStiffness(stiffness);
+HuntCrossleyRightHip.setDissipation(dissipation);
+HuntCrossleyRightHip.setStaticFriction(staticFriction);
+HuntCrossleyRightHip.setDynamicFriction(dynamicFriction);
+HuntCrossleyRightHip.setViscousFriction(viscousFriction);
+HuntCrossleyRightHip.setTransitionVelocity(transitionVelocity);
+osimModel.addForce(HuntCrossleyRightHip);
+
```

Comment:
Looks fine but extra long and tedious, can we clone and change few values instead? 10 lines per objects seems a bit long?